src/ontology/mondo-edit.obo

```diff
@@ -430439,6 +430582,410 @@ property_value: IAO:0000231 "out of scope" xsd:string
+subset: ordo_disorder
+synonym: "Brailsford disease" EXACT [Orphanet:566943]
+synonym: "Mueller-Weiss osteonecrosis of the tarsal bone" EXACT [Orphanet:566943]
+xref: Orphanet:566943 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005381 {source="https://orcid.org/0000-0001-5208-3432"} ! bone disorder
+
+[Term]
+id: MONDO:0035459
+name: idiopathic multidrug-resistant nephrotic syndrome
+subset: ordo_subtype_of_a_disorder
+xref: ICD10:N04.8 {source="MONDO:mondoSubClassOfSource", source="Orphanet:567550"}
+xref: Orphanet:567550 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005377 {source="https://orcid.org/0000-0001-5208-3432"} ! nephrotic syndrome
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/18143
+
+[Term]
+id: MONDO:0035460
+name: idiopathic steroid-resistant nephrotic syndrome with sensitivity to second-line immunosuppressive therapy
+subset: ordo_subtype_of_a_disorder
+synonym: "idiopathic steroid-resistant nephrotic syndrome with sensitivity to intensified immunosuppression" EXACT [Orphanet:567552]
+xref: ICD10:N04.8 {source="MONDO:mondoSubClassOfSource", source="Orphanet:567552"}
+xref: Orphanet:567552 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005377 {source="https://orcid.org/0000-0001-5208-3432"} ! nephrotic syndrome
+
+[Term]
+id: MONDO:0035472
+name: GJC2-related late-onset primary lymphedema
+def: "A rare genetic primary lymphedema characterized by lymphedema of all four limbs with age of onset ranging from birth to adulthood. Manifestations are of variable severity, and upper limb involvement may develop only later in the disease course. Recurrent episodes of cellulitis and skin infections are observed in severe cases. Varicose veins and venous incompetence have been reported in association." [Orphanet:568051]
+subset: ordo_disorder
+xref: Orphanet:568051 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019175 {source="https://orcid.org/0000-0001-5208-3432"} ! primary lymphedema
+
+[Term]
+id: MONDO:0035473
+name: warts-immunodeficiency-lymphedema-anogenital dysplasia syndrome
+def: "A rare primary lymphedema characterized by extensive, multisegmental lymphedema, associated with persistent, widespread infections with various genital high- and low-risk human papillomaviruses, resulting in multifocal anogenital dysplasia. Laboratory examination shows abnormalities in lymphocyte subsets, in particular CD4+ T-cells. Epidermal nevi and capillary malformations have also been reported." [Orphanet:568056]
+subset: ordo_disorder
+synonym: "disseminated warts-impaired cell-mediated immunity-primary lymphedema-anogenital dysplasia syndrome" EXACT [Orphanet:568056]
+synonym: "WILD syndrome" EXACT [Orphanet:568056]
+xref: Orphanet:568056 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019175 {source="https://orcid.org/0000-0001-5208-3432"} ! primary lymphedema
```

Comment:
'primary lymphedema' is a feature. I think the parent should instead be 'syndromic disease'

Reply:
The definition says that it is a primary lymphedema. So I think it can be a primary lymphedema (based on definition). I would still add the 'syndromic disease'

Reply:
will do